app/components/at_mention/at_mention.js

```diff
@@ -180,6 +181,15 @@ export default class AtMention extends React.PureComponent {
             onPress = isSearchResult ? onPostPress : this.goToUserProfile;
         }
 
+        if (suffix) {
+            const suffixStyle = {...styleText, color: this.props.theme.centerChannelColor};
```

Comment:
Shouldn't `textStyle` already have the colour set to `centerChannelColor`?

Reply:
nope, that is set to the color of the link, that is why we had this https://github.com/mattermost/mattermost-mobile/pull/4754#issuecomment-683920449

Reply:
Ah, I would've expected that to be `mentionStyle`, but that must be the highlighted text style.